.travis.yml

```diff
@@ -1,13 +1,18 @@
 ---
+# http://stackoverflow.com/questions/36979283/how-to-fix-travis-error-nomethoderror-undefined-method-spec-for-nilnilclass
+# https://github.com/travis-ci/travis-ci/issues/3531
+before_install:
+  - gem install bundler
+
 script: bundle exec rake
 
 rvm:
   - 1.9.3
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.5
   - rbx-2
-  - jruby
+  - jruby-9.0.5.0
```

Comment:
Same here, I believe `jruby` is supposed to automatically use the latest 9k version. I'd rather not update the Travis config file every time a new Ruby version comes out.